src/Microsoft.VisualStudio.Editors/ResourceEditor/Resource.vb

```diff
@@ -1630,11 +1630,11 @@ Namespace Microsoft.VisualStudio.Editors.ResourceEditor
         ''' file are only used to produce the .designer.cs file, which is still built against
         ''' netstandard.dll.
         ''' </remarks>
-        Private Shared Function AdjustSystemStringAssemblyQualifiedName(AssemblyQualifiedName As String) As String
-            Static SystemStringAssemblyQualifiedName As String
+        Private Shared Function AdjustAssemblyQualifiedName(AssemblyQualifiedName As String) As String
+            Static CorrectedAssemblyQualifiedName As String
```

Comment:
This doesn't make sense to be static any more, since this method now can return multiple names. In fact, the way the code is now, the method would always return the result of the first call to it.

Perhaps a Dictionary<string, Lazy<string>> might be better to use?